vispy/scene/visuals.py

```diff
@@ -228,6 +228,7 @@ def generate_docstring(subclass, clsname):
 Isoline = create_visual_node(visuals.IsolineVisual)
 Isosurface = create_visual_node(visuals.IsosurfaceVisual)
 Line = create_visual_node(visuals.LineVisual)
+Arrow = create_visual_node(visuals.ArrowVisual)
```

Comment:
Looks like it almost was -- can you alphabetize at least the simple visuals? `Compound` we might actually want at the top of the entire list since it's a bit different from the others.